ansible/roles/eos/handlers/main.yml

```diff
@@ -1,3 +1,12 @@
 - name: Restart the box
   command: /sbin/shutdown -r now "Ansible updates triggered"
 
+- name: Pause for reboot
+  pause: seconds=30
+
+- name: Wait for VM to come up
+  local_action: shell ansible -m ping -i veos {{ inventory_hostname }}
```

Comment:
>local_action [](start = 2, length = 12)

Use 'ping' module?

Reply:
Failure of ping module would cause playbook fail. Replaced the ping command with the ansible wait_for module which is designed for this kind of job.